config/routes.rb

```diff
@@ -2775,7 +2775,9 @@
   API_VERSION_REGEX = /v[\d]+(\.[\da-zA-Z]+)*(\-[\da-zA-Z]+)?/ unless defined?(API_VERSION_REGEX)
 
   # OPTIONS requests for REST API pre-flight checks
-  match '/api/*path' => 'api/base#handle_options_request', :via => [:options]
+  match '/api' => 'api/base#handle_options_request', :via => :options
+  match '/api/arbitration_rules' => 'api/arbitration_rules#options', :via => :options
+  match '/api(/*path)' => 'api/base#options', :via => :options
```

Comment:
Shouldn't we support `OPTIONS` for every URI? i.e. not just collections. Do we need to have it configurable?